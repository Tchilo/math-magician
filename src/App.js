import React from 'react';
import { Route, Switch } from 'react-router-dom';
import Calculator from './components/calculator';
import Navbar from './components/Navbar';
import Home from './components/Home';
import Quote from './components/Quote';

const App = () => (
  <>
    <Navbar />
    <Switch>
      <Route path="/App">
        <div className="App">
          <Calculator />
        </div>
      </Route>
      <Route exact path="/">
        <Home />
      </Route>
      <Route path="/Quote">
        <Quote />
      </Route>
    </Switch>
  </>
);

export default App;
