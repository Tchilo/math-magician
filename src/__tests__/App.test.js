import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

it('should render App', () => {
  const elem = renderer.create(<App />).toJSON();
  expect(elem).toMatchSnapshot();
});
