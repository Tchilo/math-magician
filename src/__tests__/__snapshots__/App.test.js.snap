// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App 1`] = `
Array [
  <nav
    className="navBar"
    data-testid="navbar"
  >
    <h2>
      Math Magicians
    </h2>
    <ul
      className="links"
    >
      <li>
        <a
          activeClassName="active-link"
          exact={true}
          href="#/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          activeClassName="active-link"
          exact={true}
          href="#/App"
          onClick={[Function]}
        >
          Calulator
        </a>
      </li>
      <li>
        <a
          activeClassName="active-link"
          exact={true}
          href="#/Quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>,
  <div
    className="Home"
  >
    <div>
      <h1
        data-testid="welcome"
      >
        Welcome to my page!
      </h1>
    </div>
    <p>
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sunt dolore corrupti adipisci? Nemo delectus, excepturi doloribus nam sint fuga quasi atque debitis inventore amet nesciunt animi a asperiores et repudiandae!
    </p>
    <p>
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sunt dolore corrupti adipisci? Nemo delectus, excepturi doloribus nam sint fuga quasi atque debitis inventore amet nesciunt animi a asperiores et repudiandae!
    </p>
  </div>,
]
`;
